include/simde/evaluate_braket/evaluate_braket.hpp

```diff
@@ -38,8 +38,12 @@ TEMPLATED_PROPERTY_TYPE_RESULTS(EvaluateBraKet, BraKetType) {
 
 #define EBK(bra, op, ket) EvaluateBraKet<type::braket<bra, op, ket>>
 
+using aos_s_e_aos  = EBK(type::aos, chemist::qm_operator::Identity, type::aos);
```

Comment:
`chemist::qm_operator::Identity` here can just be `type::s_e_type`, unless I'm missing something.

Reply:
Updated.